app/controllers/internal/events_controller.rb

```diff
@@ -135,6 +135,7 @@ def events_search_params(event_type)
       {
         type_id: event_type,
         status_ids: [pending_event_status_id],
+        start_after: DateTime.now.utc.beginning_of_day,
```

Comment:
Can we capture this in a spec? should just need to update this mock with a `.with({ type_id:....})`:

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/spec/requests/internal/events_controller_spec.rb#L41-L42